### DISCUSSION TITLE
i3bar: Move system tray to left of i3status?

### DISCUSSION THREAD

**Discussion by rlue** [i3/i3#5034] [Q&A] [answered]
I'd like to place the output of i3status to the right of the system tray, so that the clock is visible in the very upper-righthand corner of the screen. However, based on [this section of the official i3wm docs](https://i3wm.org/docs/userguide.html#_tray_padding), I suspect that's not possible.

> #### 5.9. Tray padding
>
> The tray is shown on the right-hand side of the bar...

Any chance I'm wrong about this?

**Comment by Airblader** [comment, marked as answer]
No, you're not wrong about it, this cannot be changed.

**Reply by rlue** [reply]
Thanks
